src/client/src/apps/MainRoute/widgets/spotTile/components/notifications/TileNotification.tsx

```diff
@@ -2,7 +2,7 @@ import React from 'react'
 import { styled } from 'rt-theme'
 import { Button, Icon, TileBaseStyle } from '../styled'
 
-type AccentColor = 'red' | 'green'
+type AccentColor = 'red' | 'green' | 'grey'
```

Comment:
What I am wondering is whether it is worthwhile naming these after their states, as opposed to the actual colour (e.g. info, warning, success) - so that the styling/UX can be independent from the behaviour.

Reply:
Hmm, good point. I'll take a look what is involved.